src/features/contact/contactverzoek/formulier/ContactverzoekFormulier.vue

```diff
@@ -320,7 +319,16 @@ const props = defineProps<{
 const form = ref<Partial<ContactmomentContactVerzoek>>({});
 
 const medewerker = ref<ContactVerzoekMedewerker>();
-const medewerkerFilterField = ref<string>();
+const medewerkerFilterField = computed(() => {
```

Comment:
wordt alleen gebruikt om de medewerker-search filter-field property te vullen. consistenter als we dat op dezelfde manier doen als het vullen van de filter-value property.
zoiets:

```
 <medewerker-search
        ...
        :filter-field="
          form.typeActor === ActorType.afdeling
            ? 'Smoelenboek.afdelingen.afdelingnaam'
            : form.typeActor === ActorType.groep
            ? 'Smoelenboek.groepen.groepsnaam'
            : ''
        "
        :filter-value="
          form.typeActor === ActorType.afdeling
            ? form.afdeling?.naam
            : form.groep?.naam
        "
        ...
```

Reply:
Done